testing/fuzzing/agentx_parse_fuzzer.c

```diff
@@ -53,7 +53,8 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
     netsnmp_session session;
 
     session.version = AGENTX_VERSION_1;
-    if (agentx_parse(&session, pdu, (unsigned char *)data, size) == SNMP_ERR_NOERROR) {
+    if (agentx_parse(&session, pdu, (u_char *)((uintptr_t)data), size) ==
```

Comment:
Please use NETSNMP_REMOVE_CONST() instead of this double cast.

Reply:
Thanks, fixed!